src/view/com/notifications/FeedItem.tsx

```diff
@@ -184,10 +186,24 @@ let FeedItem = ({
     action = _(msg`reposted your post`)
     icon = <RepostIcon size="xl" style={{color: t.palette.positive_600}} />
   } else if (item.type === 'follow') {
+    let isFollowBack = false
     if (
       item.notification.author.viewer?.following &&
-      gate('ungroup_follow_backs')
+      AppBskyGraphFollow.isRecord(item.notification.record)
     ) {
+      try {
+        const rkey = new AtUri(item.notification.author.viewer.following).rkey
+        const followingTimestamp = TID.fromStr(rkey).timestamp()
+        const followedTimestamp =
+          new Date(item.notification.record.createdAt).getTime() * 1000
+        isFollowBack = followedTimestamp > followingTimestamp
+      } catch (e) {
+        // For some reason the following URI was invalid. Default to it not being a follow back.
+        console.error('Invalid following URI')
```

Comment:
Just curios about this.
Console error will be never be shown in production. Wouldn't it be more efficient to pass this error to the sentry to track possible issues later on?